Framework/CMakeLists.txt

```diff
@@ -70,9 +70,13 @@ add_subdirectory (Kernel)
 include_directories (HistogramData/inc)
 add_subdirectory (HistogramData)
 
+include_directories (Beamline/inc)
+add_subdirectory (Beamline)
```

Comment:
Good. I think that we were going to do this soon anyway, and I like the name `beamline`.